tests/test_validatedentries.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) Dogeek 2020
+# For license see LICENSE
+import string
+
+import ttkwidgets.validated_entries as v_entries
+from tests import BaseWidgetTest
+import tkinter as tk
+
+
+class TestScaleEntry(BaseWidgetTest):
```

Comment:
I think the name of this test should be changed?

Reply:
True, I copy the existing tests to get the boilerplate out of the way, I guess I forgot to rename the class